core/src/trezor/wire/context.py

```diff
@@ -159,7 +161,7 @@ async def write(self, msg: protobuf.MessageType) -> None:
 CURRENT_CONTEXT: Context | None = None
 
 
-def wait(*tasks: Awaitable) -> Any:
+def wait(task: Awaitable[T]) -> Awaitable[T]:
     """
     Wait until one of the passed tasks finishes, and return the result, while servicing
```

Comment:
Docstring still refers to multiple tasks.

Reply:
9c630f56d00161e7d5209036428d5228006c87b2